projects/angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -99,6 +99,7 @@ export class ClrDatagridColumn<T = any>
   @Input('clrFilterStringPlaceholder') filterStringPlaceholder: string;
   @Input('clrFilterNumberMaxPlaceholder') filterNumberMaxPlaceholder: string;
   @Input('clrFilterNumberMinPlaceholder') filterNumberMinPlaceholder: string;
+  @Input('clrDgSortDisableUnsort') clrDgSortDisableUnsort = false;
```

Comment:
```suggestion
  @Input('clrDgDisableUnsort') disableUnsort = false;
```

The input alias is redundant (has "sort" twice) and the property name doesn't need the `clrDg` prefix.

Reply:
corrected in commit 8873189